scipy/signal/_backend.py

```diff
@@ -0,0 +1,61 @@
+import scipy._lib.uarray as ua
+from scipy.signal import _api
+
+
+__all__ = [
+    'register_backend', 'set_backend',
+    'set_global_backend', 'skip_backend'
+]
+
+
+class _ScipySignalBackend:
+    __ua_domain__ = "numpy.scipy.signal"
+
+    @staticmethod
+    def __ua_function__(method, args, kwargs):
+        fn = getattr(_api, method.__name__, None)
+
+        if fn is None:
+            return NotImplemented
+        return fn(*args, **kwargs)
+
+
+_named_backends = {
+    'scipy': _ScipySignalBackend,
+}
+
+
+def _backend_from_arg(backend):
+    if isinstance(backend, str):
+        try:
+            backend = _named_backends[backend]
+        except KeyError as e:
+            raise ValueError('Unknown backend {}'.format(backend)) from e
+
+    if backend.__ua_domain__ != 'numpy.scipy.signal':
+        raise ValueError('Backend does not implement "numpy.scipy.singal"')
+
+    return backend
+
+
+def set_global_backend(backend, coerce=False, only=False, try_last=False):
+    backend = _backend_from_arg(backend)
+    ua.set_global_backend(backend, coerce=coerce, only=only, try_last=try_last)
+
+
+def register_backend(backend):
+    backend = _backend_from_arg(backend)
+    ua.register_backend(backend)
+
+
+def set_backend(backend, coerce=False, only=False):
+    backend = _backend_from_arg(backend)
+    return ua.set_backend(backend, coerce=coerce, only=only)
+
+
+def skip_backend(backend):
+    backend = _backend_from_arg(backend)
+    return ua.skip_backend(backend)
+
+
+set_global_backend('scipy', try_last=True)
```

Comment:
Not sure fully, but these functions might get repeated across modules. We can factor out the common parts later though (dealing with concrete things is better than doing abstract thinking way before things get less hazy).

Reply:
Yes, I feel the same way. I remember you mentioned that in scipy/scipy#14407 and I guess this [comment](https://github.com/scipy/scipy/pull/14407#issuecomment-879778640) explains my thoughts on the approach since we still need different documentation.

Reply:
Maybe once all the uarray PRs land (possibly starting with scipy/scipy#14356), we can work on refactoring the common parts into a signal place which can then be used in all the modules.

Reply:
> in those functions which can also be addressed through some utility doc functions or we can completely drop module-specific examples and make these functions generalized.

I see. Well may be we can keep the module specific parts (like `Examples` section) in a common place. I don't see any harm in doing that. Like give examples from 2-3 modules in doc-string of common function (say `set_backend`).

Reply:
> we can work on refactoring the common parts into a signal place which can then be used in all the modules.

Yes, that would be way more easier than thinking of possible cases now. We don't know how things might end up in distant future.

Reply:
> Maybe once all the uarray PRs land (possibly starting with [scipy#14356](https://github.com/scipy/scipy/pull/14356)), we can work on refactoring the common parts into a signal place which can then be used in all the modules.


+1, refactoring seems an easy option later as we will be more clear which all share common behaviour.